.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
+.DS_Store
```

Comment:
A full `.gitignore` file can be created using https://www.toptal.com/developers/gitignore

Reply:
It generates a very unnecessarily verbose .gitignore that includes things we will never encounter (e.g. files that might appear in the root of a volume). It looks very amateur-ish and we seem to be manually formatting our gitignores in other repositories so this would break consistency. Adding files and paths to .gitignore is a matter of seconds, I think we really don't need this.